.eslintrc.js

```diff
@@ -28,8 +28,7 @@ module.exports = defineConfig({
     'plugin:import/warnings',
     'plugin:import/typescript',
     'plugin:promise/recommended',
-    'plugin:prettierx/standardx',
-    'plugin:prettierx/@typescript-eslint'
```

Comment:
**question:** Why do you want to remove this? :eyes: 
I didn't found something in the changelog that points to that a removal is needed or anything. Could you explain?

Reply:
Read the changelog harder.

Reply:
I don't understand
Haven't found anything here https://github.com/aMarCruz/eslint-plugin-prettierx/blob/master/CHANGELOG.md#0180---2021-04-27
And also the https://github.com/aMarCruz/eslint-plugin-prettierx/blob/master/README.md#supported-plugins doesn't explain something special about it

Help me and send me a link to the explanation, or I cannot approve this :shrug:

Reply:
https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4 

Presets removal is stated in the changelog: three presets only.

Reply:
https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4 

Presets removal is stated in the changelog: three presets only.

Reply:
So you mean this: https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101

Really bad written by their changelogs... :confused:

Reply:
> So you mean this: [aMarCruz/eslint-plugin-prettierx@9af81e2#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101](https://github.com/aMarCruz/eslint-plugin-prettierx/commit/9af81e2fd204134ca7f04e9e7baeb129874e48f4#diff-1f2167f3093e30b42f77a1caff5c4c93d44e7862e2650a8b848aa85ed1041eaaR87-R101)
> 
> Really bad written by their changelogs... 😕

I've reported that issue a month ago. I think I will make a PR if the owner does not refine the documentation and the changelog.